src/deadline/job_attachments/api/manifest.py

```diff
@@ -29,6 +36,37 @@
 """
 
 
+def _glob_files(
```

Comment:
Refactored away to share code.

Reply:
No - because this function is local only to the CLIs and only used by the CLIs, so I am not moving this to a shared code. It is shared code to process the CLI input. 

The shared function or API is `_process_glob_inputs` below.